public/app/features/alerting/unified/hooks/useSilenceNavData.ts

```diff
@@ -9,20 +9,22 @@ const defaultPageNav: Partial<NavModelItem> = {
 
 export function useSilenceNavData() {
   const { isExact, path } = useRouteMatch();
-  const [pageNav, setPageNav] = useState<Pick<NavModelItem, 'id' | 'text' | 'icon'> | undefined>();
+  const [pageNav, setPageNav] = useState<NavModelItem | undefined>();
 
   useEffect(() => {
     if (path === '/alerting/silence/new') {
       setPageNav({
         ...defaultPageNav,
         id: 'silence-new',
-        text: 'Add silence',
+        text: 'Silence alert rule',
```

Comment:
Unrelated to the changes in this PR but silences are actually applied to a list of "labels" and have no relationship with rules at all – should we make those changes in this PR?

/cc @brendamuir

Reply:
(I guess though that the _intent_ here is to silence an alert rule, so it is accurate to label it that way)